src/container/facility/FacilityOffers.tsx

```diff
@@ -3,16 +3,14 @@ import { RoomCard } from 'src/components/RoomCard';
 import { useAccommodationsAndOffers } from 'src/hooks/useAccommodationsAndOffers.tsx';
 
 export const FacilityOffers = () => {
-  const { getAccommodationById, getOffersById, offers, accommodations } =
-    useAccommodationsAndOffers();
+  const { getAccommodationById, accommodations } = useAccommodationsAndOffers();
   const params = useParams();
   const id: string = params.id as string;
   const accommodation = getAccommodationById(accommodations, id);
-  const matchedOffers = getOffersById(offers, id);
 
   return (
     <>
-      {matchedOffers.map((offer, index) => {
+      {accommodation?.offers.map((offer, index) => {
```

Comment:
is it possible to get accommodation without `offers`?

Reply:
nope ,It could be empty array.

Reply:
if it `nope ,It could be empty array` you should not put question mark there

Reply:
please use a question mark only as a `last chance`

Reply:
its for accommodation not to be `null`

Reply:
![Screenshot 2022-08-22 at 13 55 34](https://user-images.githubusercontent.com/47505482/185915471-df3cfc31-d0e4-48e1-b84a-ca5b7b1d1e27.jpg)